### ISSUE TITLE
Since paper update, Jobs gets hung on startup; prevents server from starting

### ISSUE THREAD

**Issue by davidp5118** [Zrips/Jobs#1380]
We have had jobs working smoothly on our server for about 6 months now. When I went to bed last night everything was working fine. This morning we did a quick server restart. Nothing was updated or changed. When the server tried to start, it got hung up on jobs and would just hang on jobs until the whole server would crash. 

We tried it with the last 2 jobs releases, and same thing. 

The only thing we have that auto-updates is paper, which did update 2 days ago. This is our first restart since then. When we quarantine jobs, the server started.

TLDR: the most recent update of paper seems to have broken our jobs plugin.

**Comment by davidp5118** [comment]
FYI--issue was connecting to hosting site
